include/common_robotics_utilities/ros_conversions.hpp

```diff
@@ -4,96 +4,118 @@
 
 #include <Eigen/Geometry>
 #include <common_robotics_utilities/math.hpp>
+
+#if COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 2
+#include <geometry_msgs/msg/point_stamped.hpp>
+#include <geometry_msgs/msg/pose_stamped.hpp>
+#include <geometry_msgs/msg/transform_stamped.hpp>
+#elif COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 1
 #include <geometry_msgs/PointStamped.h>
 #include <geometry_msgs/PoseStamped.h>
 #include <geometry_msgs/TransformStamped.h>
+#else
+#error "Undefined or unknown COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION"
+#endif
```

Comment:
We may have to move away from this and have separate `include` files too. With CMake it's easy to export compile definitions, but Bazel requires them to be set for the library and for each of its dependencies...

Reply:
Nevermind, using the `defines` attribute in `cc_library` rules does the trick.